pkgs/top-level/all-packages.nix

```diff
@@ -5427,6 +5427,7 @@ with pkgs;
 
   # TeX Live; see https://nixos.org/nixpkgs/manual/#sec-language-texlive
   texlive = recurseIntoAttrs (callPackage ../tools/typesetting/tex/texlive { });
+  inherit (texlive.schemes) texliveBasic texliveContext texliveFull texliveGust texliveInfraonly texliveMedium texliveMinimal texliveSmall texliveTetex;
```

Comment:
Do you mean why `texliveBasic` instead of `texliveSchemeBasic`? I think I was just parroting `python3Full`.

Come to think of it, Debian uses `texlive-full`, Fedora uses `texlive-scheme-full`. Our combined yielded `texlive-combined-full`. So no particular reason, really. Although if we do multiple years, `texlive2023SchemeFull` gets long.

Sidenote, `name` is always `texlive-env-2022` (should be `texlive-2022-env` actually!). Requested packages get listed in `meta.longDescription`.

(I suppose we agree that top level schemes is better than the current `callPackage { texLive = texlive.combined.scheme-small; }` approach. I'd rather have packages work like `asymptote` in this PR.)